client/post-editor/media-modal/index.jsx

```diff
@@ -210,7 +210,7 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorDone( error, blob, imageEditorProps ) {
+	onImageEditorDone( error, blob, imageEditorProps, media ) {
```

Comment:
I removed my comment. Here we go again:
I thought that when I did this implementation. IMHO `image-editor` shouldn't be connected with a media post since it could reduce the usage scope.
What happens if we wanna use the editor in other pieces where passing a media post doesn't make sense such as user avatar, etc.
could we extend `image-editor` creating a new component named `media-editor` for instance?